R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -502,20 +502,21 @@ test_that("jsonRDD() on a RDD with json string", {
 })
 
 test_that("test tableNames and tables", {
+  previous_count <- length(tableNames())
   df <- read.json(jsonPath)
   createOrReplaceTempView(df, "table1")
-  expect_equal(length(tableNames()), 1)
+  expect_true(length(tableNames()) == 1 + previous_count)
```

Comment:
Thank you, @felixcheung .
This PR introduces a group of predefined system views. It will be read-only and persistent.